### ISSUE TITLE
Trading prices are not updated

### ISSUE THREAD

**Issue by furkancanturk** [yasserfarouk/scml#66]
Hi,
self.awi.trading_prices gives the list of catalog prices each step. It seems it is not updated although product flow occurs.

**Comment by yasserfarouk** [comment]
OneShotWorld or SCML2021World?

**Comment by furkancanturk** [comment]
It is SCML2021World.
I keep the log of trading prices each step and each price remains same throughout the run. I know that all these prices are catalog prices considering my input and output catalog prices.

**Comment by yasserfarouk** [comment]
Ok I will try to reproduce the issue and return to you.

**Comment by yasserfarouk** [comment]
We reproduced the issue. It should be fixed now in 230d6d1

To try it. please:

```bash
pip uninstall scml
pip install -U git+https://www.github.com/yasserfarouk/scml
```

We added a test (see the commit above) that checks the trading price and confirms that it changes when there is trade at a price different from the catalog price. It seems to pass.

**Comment by furkancanturk** [comment]
It is fixed!
